### DISCUSSION TITLE
Zoom goes to upper left corner and hotkeys do not work half of the time

### DISCUSSION THREAD

**Discussion by Traskers** [tryptech/obs-zoom-and-follow#126] [Q&A]
I have no idea what I am doing. I just spent the past two hours getting error after error about different issues, and after I've got it all working on OBS 29.1.3 and Python 3.10.9, the hotkey didn't work. I then found out that there were two hotkeys made, and I've tried both. Randomly, one of the hotkeys worked, but it started zooming in really far on the top left corner, even clipping off of the monitor screen. About half of the time I pressed the hotkey, the screen would move like 10 pixels up without zooming back out. So, I deleted the capture source I was using and replaced it with a new one. After repeating this process around three more times, I just gave up. Is this an issue currently being worked? I haven't been able to find any answers for this issue.

**Comment by kiteliongames** [comment]
Had the same exact issue. Chalked it up to having multiple monitors.

**Comment by albertocabasvidani** [comment]
Do you mean that the zoom is just cropping the top left corner? Because this is happening to me. And I can't toggl it off
